app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, file_in, file_out = command.split(" ")
+    if mv != "mv":
+        return
+    os.makedirs("/".join(file_out.split("/")[:-1]))
+    if file_out.split("/")[-1] == "":
+        raise FileNotFoundError
```

Comment:
do you need this check ? if the command will be like this "mv file.txt first_dir/second_dir/third_dir" will it work ?